recipe/meta.yaml

```diff
@@ -63,9 +77,14 @@ requirements:
     - setuptools
 
 test:
+  requires:
+    - pytest
+  source_files:
+    - tests/
   commands:
     - test -f "${PREFIX}/bin/openmc"
     - openmc --version
+    - pytest -v tests/unit_tests/dagmc/  # [dagmc == 'dagmc']
```

Comment:
What if you run these as part of the build script?

Reply:
I think the files are missing or in a different path...

Reply:
I am not sure where the test files are hiding but I'm looking for them now. This method [works on another package ](https://github.com/conda-forge/neutronics_material_maker-feedstock/blob/fbe668ed8a9417418e21e1e37b16990dcee4bd93/recipe/meta.yaml#L32-L35)so I'm not quite sure why the folder structure is different here. I hope to get this working soon

Reply:
This works in the other package because all the files for the tests are installed during the build process.  

OpenMC's test files are in the source tree, and available during a build step, but are NOT installed.  

When conda-forge invokes the `test:` task, it starts with a clean system, conda-installs the package in question and then runs the commands in the tests.  When OpenMC is conda installed during this phase, it includes no tests because they were never installed as part of the package.

Reply:
Thanks for explaining this gonuke

I started adding this to the end of the build.sh
```
if [[ -n "$dagmc" && "$dagmc" != "nodagmc" ]]; then
  pytest -v tests/unit_tests/dagmc/test.py
fi
```
But now I think it might be best to leave out the pytest part as this would require all the run time dependences at build time.